bin/run_dwelling.py

```diff
@@ -47,8 +47,11 @@
     #     'external_radiation_method': 'linear',
     #     'internal_radiation_method': 'linear',
     #     'reduced_states': 7,
+    #     'save_matrices': True,
```

Comment:
I think we should keep it. The goal is to show some of the options that users might want to enable. It's commented out so it would actually get disabled unless someone uncomments it.

Reply:
You're right, I don't know why I thought it wasn't commented out. OK all good then